src/lib.rs

```diff
@@ -1,5 +1,7 @@
 #![doc = include_str!("../README.md")]
 
+extern crate core;
+
```

Comment:
Was this added by RustRover? Not sure what this does

Reply:
Good catch! I added a commit that removed this line. It is completely unnecessary and was added by rustrover